student_class.rb

```diff
@@ -0,0 +1,14 @@
+require './person_class'
+# Student class
+class Student < Person
+  attr_accessor :classroom
+
+  def initialize(classroom, _age, _name = 'Unknown', parent_permission = 'true')
```

Comment:
- [ ] Here is the same as above because if we create an object from students it will return nil so it's better to let the program read the predetermine parameter otherwise it'll be nil.